crates/autopilot/src/solvable_orders.rs

```diff
@@ -907,12 +926,78 @@ mod tests {
         },
     };
 
+    #[tokio::test]
+    async fn filters_tokens_without_native_prices_with_timeout() {
```

Comment:
This and possibly other tests can be dropped if we remove the selective native price filtering mentioned in another thread.